### DISCUSSION TITLE
how to run multiple sql query in one orientdb query statement

### DISCUSSION THREAD

**Discussion by hgftrdw45ud67is8o89** [orientechnologies/orientdb#10119] [Q&A] [answered]
I can find the sql query need to CRUD vertex in https://orientdb.org/docs/3.0.x/sql/SQL-Create-Vertex.html
I tried some code to setup *sql graph query for orientdb*

```
String statement = "CREATE CLASS Person EXTENDS V;CREATE VERTEX Person";//this query is wrong?
				OResultSet rs = database.query(statement);
				while(rs.hasNext()){
				  OResult row = rs.next();
				}
				rs.close();
```

it returns>
com.orientechnologies.orient.core.sql.OCommandSQLParsingException: Error parsing query:
CREATE CLASS Person EXTENDS V;CREATE VERTEX Person
                                   ^
Encountered " <CREATE> "CREATE "" at line 1, column 31.

**Comment by andrii0lomakin** [comment]
Hi, please use  `\n` instead. 
We use a line separator. But we should consider to use ; too.

**Reply by hgftrdw45ud67is8o89** [reply]
sadly the newline doesn't work

```
CREATE VERTEX
 CREATE CLASS Person EXTENDS V
      ^
CREATE VERTEX Person CONTENT { "name" : "Jay", "surname" : "Miner" }
Encountered " <CREATE> "CREATE "" at line 2, column 2.
Was expecting one of:
```

**Reply by tglman** [reply, marked as answer]
Hi, the API for running multiple statement is :

```java
session.execute("sql", query, args);
```
and not the `session.query`

Regards

**Reply by hgftrdw45ud67is8o89** [reply]
tested an it worked thanks.
